packages/plugin-core/src/commands/Doctor.ts

```diff
@@ -67,6 +96,71 @@ export class DoctorCommand extends BasicCommand<CommandOpts, CommandOutput> {
         });
         break;
       }
+      case DoctorActions.CREATE_MISSING_LINKED_NOTES: {
+        const notes = engine.notes;
+        // pick out wild wikilinks
+        let wildWikiLinks = [] as DLink[];
+        _.forEach(notes, (note) => {
+          const links = note.links;
+          if (_.isEmpty(links)) {
+            return false;
+          }
+          const wsRoot = DendronWorkspace.wsRoot();
+          wildWikiLinks = wildWikiLinks.concat(
+            _.filter(links, (link) => {
+              if (link.type != "wiki") {
```

Comment:
since its javascript, we want to use strict equality check `!==` (see [here](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons))